salt/cloud/clouds/linode.py

```diff
@@ -993,6 +993,58 @@ def get_password(vm_):
     )
 
 
+def _decode_linode_plan_label(label):
+    """
```

Comment:
I know this is picky, but can you change these to single quotes instead of double? That's what we prefer according to the [style guide](https://docs.saltstack.com/en/latest/topics/development/conventions/style.html).